components/automate-ui/src/app/components/table/table-body/table-body.component.scss

```diff
@@ -0,0 +1,5 @@
+@import '../table.component';
+
+:host {
+  display: table-row-group;
```

Comment:
The table* values for display provide the semantics for creating a table without using `<table>`.